extensions/amp-story/1.0/amp-story.js

```diff
@@ -336,6 +337,11 @@ export class AmpStory extends AMP.BaseElement {
     /** @private @const {!../../../src/service/viewer-interface.ViewerInterface} */
     this.viewer_ = Services.viewerForDoc(this.element);
 
+    /** @private @const {?AmpStoryViewerMessagingHandler} */
+    this.viewerMessagingHandler_ = this.viewer_.isEmbedded()
```

Comment:
Curious, why do we have to check if viewer is embedded?

Reply:
If it's not there's no one to send any message, so we don't even set the listeners.